.github/workflows/validate.yml

```diff
@@ -1,5 +1,16 @@
 name: validate
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      [
+        "+([0-9])?(.{+([0-9]),x}).x",
+        "master",
+        "next",
+        "next-major",
+        "beta",
+        "alpha",
+      ]
+  pull_request: {}
```

Comment:
Ah, I didn't know this was the format! I would have thought that just `pull_request` was enough